docs/news/posts/2025-10-06-migration-overview.md

```diff
@@ -0,0 +1,219 @@
+
+#### Changes to Mainline Partitions
+
+On GPFS 4, all nodes in the interactive/express/short/medium/long (mainline) partitions have Intel processors while the `amd-hdr100` nodes have AMD processors. On GPFS 5, the mainline partitions use nodes that were previously migrated from the `amd-hdr100` partition and so now use AMD CPUs.
+
+In rare cases, tools compiled on on Intel processor can cause `Illegal Instruction` errors when run on AMD CPUs. If your jobs were submitted to one of the mainline partitions and show this error after migration, please [contact support](../../index.md#how-to-contact-us). As the migration continues, the standard Intel nodes will also be migrated from GPFS 4 to GPFS 5 and added to the mainline partitions.
+
+#### Compute Migration
+
+To best accommodate workloads on both clusters during data migration, compute nodes will be migrated in two stages: at 50% and 100% data migration. Due to hardware networking and rack constraints, we have limited options as far as which nodes we can move at which times. See below for a list of which nodes will be migrated to GPFS 5 at which time.
+
+**50% Migration Completion**:
+
+- All pascalnodes (18 total nodes on GPFS 5, 72 available P100s)
+- 10 amperenodes (15 total on GPFS 5, 30 available A100s)
+
+**100% Migration Completion**:
+
+- 5 amperenodes (20 total on GPFS 5)
+- All largemem nodes (14 total on GPFS 5)
+
+#### Effects on Queue Times
+
+During the migration, be aware that job wait times are expected to increase based on when you are migrated relative to the [compute migration](#compute-migration). For instance, users migrated prior to the initial compute migration will experience longer wait times due to initially lower compute capacity on GPFS 5 while users still on GPFS 4 will experience somewhat reduced wait times due to fewer jobs competing for resources on that platform. This will flip after the initial compute migration where users on GPFS 5 will have reduced wait times from the influx of compute nodes, and users on GPFS 4 will have longer wait times.
+
+**This will be especially apparent for jobs on pascalnodes and amperenodes.** Prior to the initial compute migration, there will be a 100%:0% split of pascalnodes and 75%:25% split of amperenodes on GPFS4 and GPFS 5, respectively. After the initial compute migration, the splits will be 0%:100% for pascalnodes and 25%:75% for amperenodes on GPFS 4 and GPFS 5, respectively. We understand this will have a large impact on job throughput for individual users and will endeavor to complete the migration swiftly to prevent undue burden on our community. Thank you for understanding.
+
+### Directory Paths
+
+We have reorganized our GPFS 5 filesets to improve file access performance specifically for our network scratch space. As part of this, we have changed the `/data` drive to `/gpfs` and have brought `/scratch` under it.
+
+**Path conversions from GPFS 4 to GPFS 5**:
```

Comment:
Probably can remove this section given our discussions around bind mounts and retaining a stable file namespace.